# ROS config
heartbeat_duration:   1.0  # secs
PID_rate:             10.0 # Hz

# Thrust controller configuration
motor_cmd_limit:      2.0
thrust_config:        'H'
lateral_scale_factor: 0.7
neg_scale_factor:     0.666666
priority_yaw_range:   1.570796 

# Linear PID controller gains
lin_Kp: 1.0
lin_Ki: 0.0
lin_Kd: 0.0

# Angular PID controller gains
ang_Kp: 1.0
ang_Ki: 0.0
ang_Kd: 0.0

# General PID configuration
use_sim_time: true
max_integral: 5.0